tdrs-backend/clamav-router/nginx.conf

```diff
@@ -0,0 +1,20 @@
+events { worker_connections 1024;
+}
+
+
+http{ 
+    server {
+        listen {{port}};
+        location /scan {
+            proxy_pass                      http://tanf-prod-clamav-rest.apps.internal:9000/scan;
+            proxy_pass_request_headers      on;
+        }
+    }
+    server {
+        listen 9000;
+        location /scan {
```

Comment:
This opens a route to clamav prod, we might want to add a note somewhere for revising this later on. Although all traffic is from cloud.gov space, but they are from a space out of prod